core/types/gen_header_json.go

```diff
@@ -134,5 +137,8 @@ func (h *Header) UnmarshalJSON(input []byte) error {
 	if dec.Nonce != nil {
 		h.Nonce = *dec.Nonce
 	}
+	if dec.BaseFee != nil {
+		h.BaseFee = (*big.Int)(dec.BaseFee)
```

Comment:
Also set `Eip1559` to `true` if it is still there

Reply:
added